src/Sylius/Bundle/ImportExportBundle/Form/Type/Writer/XlsWriterType.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\ImportExportBundle\Form\Type\Writer;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+/**
+ * Xls writer type.
+ *
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+class XlsWriterType extends AbstractType
```

Comment:
In fact, I don't know is this writer implemented. I suppose no. So this FormType and every other files could be removed. Or we could just implement XlsWriter.